addins/Cake.Unicorn.yml

```diff
@@ -1,9 +1,8 @@
 Name: Cake.Unicorn
 NuGet: Cake.Unicorn
-Prerelease: false
 Assemblies:
 - "/**/Cake.Unicorn.dll"
-Repository: https://github.com/georgechang/Cake.Unicorn
+Repository: https://www.nuget.org/packages/Cake.Unicorn/
```

Comment:
@Jericho this seems wrong.

Reply:
I agree but this value was taken straight from the nuspec and since this project was not move to the cake-contrib organization, there is no way for the audit tool to reverse engineer the correct URL.

Reply:
Maybe not fallback to nuget if an existing valid repo exists?

Reply:
Where is this existing valid URL stored? I don't see it in the nuspec.

Reply:
Could only be in the yaml, but feels wrong to remove a correct url with a fallback one.

Reply:
In my opinion, what is wrong is for the addin author to include an incorrect URL in their nuspec.

Reply:
Bottom line is: the audit tool uses nuspec from NuGet.org as the "source of truth". If the nuspec contains incorrect data, the audit tool will generate yaml files with incorect values.

Reply:
I'd like to correct what I previously stated: this URL was not taken from the package nuspec. The issue is that the URL is **missing** from the nuspec and therefore the audit tool used the nuget.org URL. Here is the full content of this package nuspec:
```
<?xml version="1.0" encoding="utf-8"?>
<package xmlns="http://schemas.microsoft.com/packaging/2013/05/nuspec.xsd">
  <metadata>
    <id>Cake.Unicorn</id>
    <version>1.0.14</version>
    <authors>Cake.Unicorn</authors>
    <owners>Cake.Unicorn</owners>
    <requireLicenseAcceptance>false</requireLicenseAcceptance>
    <description>Package Description</description>
    <dependencies>
      <group targetFramework=".NETFramework4.6.1">
        <dependency id="Cake.Core" version="0.30.0" exclude="Build,Analyzers" />
        <dependency id="MicroCHAP" version="1.2.2.2" exclude="Build,Analyzers" />
      </group>
    </dependencies>
    <frameworkAssemblies>
      <frameworkAssembly assemblyName="System.Net.Http" targetFramework=".NETFramework4.6.1" />
      <frameworkAssembly assemblyName="System.Web" targetFramework=".NETFramework4.6.1" />
    </frameworkAssemblies>
  </metadata>
</package>
```
So the issue is that the author did not include the project URL.

Reply:
What's bizarre is that the [nuspec in the repo](https://github.com/georgechang/Cake.Unicorn/blob/master/src/Cake.Unicorn/Cake.Unicorn.nuspec#L9) is totally different than the nuspec published to nuget. So we would have to contact the author and point out the discrepancies to him (in addition to `ProjectUrl`, the `Author`, the `Owner` and `Description` properties in the published nuspec do not match what's in the nuspec in the repo).

Reply:
I raised an [issue](https://github.com/georgechang/Cake.Unicorn/issues/5) in the addin repo asking the author to fix the published nuspec.